code/style.css

```diff
@@ -1,4 +1,288 @@
+  margin: 10px;
+}
+
+.small-text-big-news {
+  margin-top: 0px;
+  font-size: 20px;
+  margin: 0px 10px;
+}
+
+h3 {
+  font-size: 24px;
+  text-transform: uppercase;
+  margin: 10px 5px 5px 5px;
+}
+
+p {
+  margin: 0px 5px 25px 5px;
+}
+
+p:hover {
+  text-shadow: 1px 1px grey;
+}
+
+.more-news {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.news-card {
+  background-color: whitesmoke;
+  width: 100%;
+  padding-top: 5px;
+  margin-bottom: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+img {
+  width: 80%;
+}
+
+.news-card:hover {
+  background-color: white;
+  border-top: 2px solid coral;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: whitesmoke;
+  padding: 30px;
+  animation-name: disco-footer;
+  animation-duration: 3s;
+  animation-iteration-count: infinite;
+}
+
+.foot-text1, .foot-text2 {
+  padding-top: 25px;
+}
+
+.button {
+  padding: 8px 16px;
+  border-radius: 25px;
+  border: none;
+  background-color: coral;
+  color: whitesmoke;
+  font-size: 15px;
+}
+
+a {
+  text-decoration: none;
+  color: rgb(78, 77, 77);
+}
+
+/*tablet version*/
+@media (min-width: 668px) and (max-width: 1023px) {
+  h1 {
+    font-size: 50px;
+    left: 150px;
+  }
+
+    .fa-bars {
+    display: inline;
+  }
+
+  .bars {
+    display: flex;
+    align-items: center;
+  }
+
+    .big-news {
+      flex-direction: row;
+      justify-content: space-evenly;
+      align-items: center;
+      width: 100%;
+    }
+
+    .gotland-image {
+      width: 50%;
+    }
+    
+    .big-news:hover {
+      text-shadow: 1px 1px grey;
+    }
+
+    .big-news-text:hover {
+      background-color: whitesmoke;
+    }
+
+    .more-news {
+      flex-direction: row;
+      flex-wrap: wrap;
+      justify-content: space-evenly;
+      width: 100%;
+      padding-left: 2px;
+      padding-right: 2px;
+    }
+
+    .news-card {
+      width: 48%;
+    }
+
+    img {
+      width: 80%;
+    }
+
+    .button {
+      position: absolute;
+      left: 40%;
+    }
+
+    .foot-text1, .foot-text2 {
+      display: none;
+    } 
+}
+
+/*desktop version*/
+@media (min-width: 1024px) {
+  h1 {
+    font-size: 50px;
+    left: 150px;
+  }
+
+  .fa-bars {
+    display: none;
+  }
+
+  .nav-text1, .nav-text2 {
+    display: inline;
+  }
+
+  .big-news {
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+  }
+
+  .more-news {
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+  }
+
+  .news-card {
+    width: 23%;
+  }
+
+  .big-news:hover {
+    text-shadow: 1px 1px grey;
+  }
+
+  .big-news-text:hover {
+    background-color: whitesmoke;
+  }
+
+  .news-card:hover {
+    border: 1px solid grey;
+    padding: 3px;
+    box-shadow: 7px 7px 4px grey;
+    float: left;
+    -webkit-transform: rotate(6deg);
```

Comment:
This hovering effect is being made so clear and awesome! I cannot stop clicking on them. 
One small question on `-webkit-transform:` code, do you know when it works in specific? I had a lot of cases where only that would work for width element in my code.